
Due to MODULEPATH changes, the following have been reloaded:
  1) openmpi/2.1.1

Python 3.5.4
Using TensorFlow backend.
/home/amourav/tensorflow/lib/python3.5/site-packages/skimage/transform/_warps.py:105: UserWarning: The default mode, 'constant', will be changed to 'reflect' in skimage 0.15.
  warn("The default mode, 'constant', will be changed to 'reflect' in "
/home/amourav/tensorflow/lib/python3.5/site-packages/skimage/transform/_warps.py:110: UserWarning: Anti-aliasing will be enabled by default in skimage 0.15 to avoid aliasing artifacts when down-sampling images.
  warn("Anti-aliasing will be enabled by default in skimage 0.15 to "
Traceback (most recent call last):
  File "./predict.py", line 104, in <module>
    args.modelPath)
  File "./predict.py", line 80, in main
    model = load_model(modelPath)
  File "/home/amourav/tensorflow/lib/python3.5/site-packages/keras/engine/saving.py", line 417, in load_model
    f = h5dict(filepath, 'r')
  File "/home/amourav/tensorflow/lib/python3.5/site-packages/keras/utils/io_utils.py", line 186, in __init__
    self.data = h5py.File(path, mode=mode)
  File "/home/amourav/tensorflow/lib/python3.5/site-packages/h5py/_hl/files.py", line 394, in __init__
    swmr=swmr)
  File "/home/amourav/tensorflow/lib/python3.5/site-packages/h5py/_hl/files.py", line 170, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 85, in h5py.h5f.open
OSError: Unable to open file (unable to open file: name = './modelOutput/compareModels3/Xception/Xception_2019-06-09_10_33_/Xception.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
